eslint_ecma5.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 5,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": false
+    }
+  },
+  "rules": {
+    // Inspired by airbnb rules: https://github.com/airbnb/javascript/blob/master/linters/SublimeLinter/SublimeLinter.sublime-settings
+    // Force all variable names to use either camelCase style or UPPER_CASE with underscores.
+    "camelcase": [2, {"properties": "always"}],
+
+    // Prohibit use of == and != in favor of === and !==.
+    "eqeqeq": 2,
```

Comment:
in a case of consistency, I think it is better to keep one convention in actions naming, (there is 
`"eqeqeq": 2` and after that `"indent": ["error", "tab"]`). I think that the second notation is more readable, so `"eqeqeq": "error"`